src/pages/starter-kit/checkout/tax-reference.md

```diff
@@ -258,6 +258,14 @@ There are two GraphQL queries to check taxes applied by the tax integration, `ca
 
 To check the taxes applied to the cart, you can use the [`getCart`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/queries/cart/) query to retrieve the [`cart`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-Cart)/[`prices`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-CartPrices)/[`applied_taxes`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#definition-CartPrices) field. This field contains information about the taxes applied to the cart.
 
+>[!TIP]
+>
+>To enable full tax breakdown, set `Stores > SETTINGS > Configuration > SALES > Tax > Shopping Cart Display Settings > Display Full Tax Summary` to `Yes`
```

Comment:
Do we need to define what a "full tax breakdown" is?

Reply:
@jhadobe The tax breakdowns that are available in Cart, Order, etc from Admin, Luma, GraphQL that Adobe Commerce instance supports. This is the detail wiki page that you can refer to: https://wiki.corp.adobe.com/display/ACCSS/Tax+Breakdown%2C+applied+taxes+and+beyond
Please let us know if you have any suggestions to groom the words.

Reply:
thanks for the context @kuflower, i would suggest something like this:
```
To enable full tax breakdown, which displays each tax separately instead of combining them, set ...
```